Makefile

```diff
@@ -216,3 +216,11 @@ build_localnet_validator:
 
 protofiles:
 	bash ./scripts/gogenerate.sh
+
+# start a log aggregation toolset(Promtail->Loki->Grafana)
+# needs docker and docker-compose
+debug-start-log-aggregator:
```

Comment:
feels a bit long to type what do you think of 
debug-start-logagg (or simpy debug-start-log) 

and similar naming for `stop`

Reply:
agreed

Reply:
+ added these 2 commands to the `help` command

Reply:
+ one more command `debug-restart-log` - just do `debug-restart-log: debug-stop-log debug-start-log`